strr-api/src/strr_api/__init__.py

```diff
@@ -64,31 +64,31 @@ def create_app(environment: Config = Production, **kwargs) -> Flask:
     CORS(app)
     app.config.from_object(environment)
 
-    # Configure Sentry
-    if dsn := app.config.get("SENTRY_DSN", None):
-        sentry_sdk.init(
-            dsn=dsn,
-            integrations=[FlaskIntegration()],
-            release=f"strr-api@{get_run_version()}",
-            send_default_pii=False,
-            environment=app.config.get("POD_NAMESPACE", "unknown"),
-        )
-
     db.init_app(app)
-
-    if not app.config.get("TESTING", False):
+    if app.config.get("POD_NAMESPACE", "production") == "migration":
```

Comment:
why would we be running in a migration namepsace?
POD_NAMESPACE matches a kubernetes OS value, which we don't use in knative.